README.md

```diff
@@ -68,6 +68,11 @@
 
 - [Maze generator](https://github.com/Dannark/FlameExamples) - List of algorithm examples to solve problems like Maze generation and more. By [Dannark](https://github.com/Dannark/)
 
+### YouTube videos
+
+<!--lint ignore double-link-->
+- [Dino Run - Flutter Game Development Series](https://youtube.com/playlist?list=PLiZZKL9HLmWOmQgYxWHuOHOWsUUlhCCOY) -  An easy to follow tutorial on how to make 2D mobile game in Flutter using Flame engine. By [DevKage](https://www.youtube.com/devkage)
```

Comment:
This could be a dded under the "Other Articles & Tutorials" instead of under its own headline

Reply:
Made the change in latest commit.